src/devices/Bmxx80/Bme680.cs

```diff
@@ -450,7 +450,11 @@ protected override void SetDefaultConfiguration()
             FilterMode = Bme680FilteringMode.C0;
 
             _bme680Calibration = (Bme680CalibrationData)_calibrationData;
-            TryReadTemperature(out var temp);
+            if (!TryReadTemperature(out var temp))
+            {
+                temp = Temperature.FromDegreesCelsius(22.0);
```

Comment:
I'm confused - 22C if we can't read? would it make sense to throw or something?

Reply:
I just selected a reasonable default, previously the return value was ignored, which made the variable undefined (note that with the usage of UnitsNet, the value is not necessarily Zero, but really unusable). 

I'm not sure I want to throw here, as it would eventually cause the ctor to throw, just because an initial temperature couldn't be read.

Reply:
I'd rather have this something more obvious it's invalid, i.e. FromKelvin(0) or something

Reply:
It should be a reasonable ambient temperature. It is used to pre-configure the heater of the gas sensor. A small error will not matter much, as that heater is heats up to 250-350 °C, but 0°K is very far off (and unlikely to be accurate, unless the world has ended). The spec says to use the temperature measurement or a hardcoded value. 

Maybe I should make it an (optional) parameter?

Reply:
@krwq: Ping. I'm not sure how to best proceed here.

Reply:
Updated with an argument